docs/content/develop/breaking-changes/breaking-changes.md

```diff
@@ -77,6 +77,9 @@ For more information, see
     if the change will destroy and recreate the resource due to changing an immutable value.
     Default changes in the provider are comparable in impact to default changes in an API,
     and modifying examples and modules may achieve the intended effect with a smaller blast radius.
+  * Adding a default value to an optional, immutable field can cause Terraform to propose destroying and
```

Comment:
New default values on existing immutable fields is covered in the preceding paragraph, so I think you could leave it out here.

What would you think about stating this more generally - for example something like "adding a new field with a default value is considered a breaking change if it can result in a diff on a user's first apply after upgrading"? It could go before the "when a default is changed" so that paragraph (maybe with light modification?) can apply to all cases when a new default causes a diff, whether the field is old or new.

Ideally we could also say somewhere "... and here's a link to how to add new fields with defaults safely" (which would involve figuring out the API default and maybe writing a state upgrader in some cases?) but I think that would likely be a part of the permadiff best practices doc from https://github.com/GoogleCloudPlatform/magic-modules/pull/9884 so it wouldn't be possible to link to yet.